### DISCUSSION TITLE
Cloud Stack

### DISCUSSION THREAD

**Discussion by SebastianStehle** [RaythaHQ/raytha#173] [Q&A]
Hi,

it is very interesting. What do you use as a cloud stack? Have you built everything around billing yourself?

Sebastian

**Comment by apexdodge** [comment]
Hi, thanks for the question and Merry Christmas.

On the cloud platform, websites are provisioned in Azure's East US region. We intend to support more regions in the future. Each website is provided:

- As a pod in an AKS Cluster. Higher paid tiers get redundant pods, auto-scaling, and increased dedicated cpu and memory.
- A database in an Azure SQL Elastic Pool
- A container in Azure Blob Storage

Universally, we are using:

- Sendgrid for email.
- Cloudflare for DNS, Custom Domains, and CDN
- Azure Traffic Manager

Everying around billing is custom built on top of Stripe Billing's API.

Hope that helps. Cheers!

**Reply by SebastianStehle** [reply]
So you have built it yourself? Great. I was thinking about something similar for Squidex (My CMS) but so far I only have a shared deployment

**Reply by apexdodge** [reply]
Yes, there's a trade off doing shared v dedicated resources. Unit economics are much worse with dedicated compute, but there is better security and reliability. It is a hard decision.
